benches/benches/block_target_gas_set/contract.rs

```diff
@@ -1,20 +1,741 @@
+    //     "contract/srwq",
+    //     group,
+    //     instructions,
+    //     script_data.clone(),
+    //     &service,
+    //     contract_id,
+    //     &rt,
+    //     &mut rng,
+    // );
+
+    // sww
+
+    {
+        let contract = vec![op::sww(RegId::ZERO, 0x29, RegId::ONE), op::ret(RegId::ZERO)];
+        let instructions = call_contract_repeat();
+        replace_contract_in_service(&mut service, &contract_id, contract);
+        run_with_service(
+            "contract/sww",
+            group,
+            instructions,
+            script_data.clone(),
+            &service,
+            contract_id,
+            &rt,
+            &mut rng,
+        );
+    }
+
+    // swwq
+
+    // TODO: This is under-costed, so it runs too long and will complete before running out of gas
+    // let size = 2620_u32;
+    // // Copy value stored at $rC to the state starting at 0x13
+    // let contract = (0..2620)
+    //     .map(|x| x * size)
+    //     .map(|x| vec![op::movi(0x13, x), op::swwq(0x13, 0x29, 0x14, 0x15)])         .flatten()
+    //     .collect();
+    // let gas = 100_000;
+    // let value = 2000;
+    // let instructions = vec![
+    //     op::gtf_args(0x10, 0x00, GTFArgs::ScriptData),
+    //     op::addi(0x11, 0x10, ContractId::LEN.try_into().unwrap()),
+    //     op::addi(0x11, 0x11, WORD_SIZE.try_into().unwrap()),
+    //     op::addi(0x11, 0x11, WORD_SIZE.try_into().unwrap()),
+    //     op::movi(0x12, gas),
+    //     op::movi(0x14, value),
+    //     op::movi(0x15, size),
+    //     op::call(0x10, RegId::ZERO, 0x11, 0x12),
+    // ];
+    // replace_contract_in_service(&mut service, &contract_id, contract);
+    // run_with_service(
+    //     "contract/swwq",
+    //     group,
+    //     instructions,
+    //     script_data.clone(),
+    //     &service,
+    //     contract_id,
+    //     &rt,
+    //     &mut rng,
+    // );
+
+    // time
+
+    {
+        run(
+            "contract/time",
+            group,
+            vec![
+                op::movi(0x10, 0),
+                op::time(0x11, 0x10),
+                op::jmpb(RegId::ZERO, 0),
+            ],
+            vec![],
+        );
+    }
+
+    // tr
+
+    {
+        let contract = vec![op::tr(0x15, 0x14, 0x15), op::ret(RegId::ZERO)];
+        let mut instructions = setup_instructions();
+        instructions.extend(vec![
+            op::movi(0x15, 2000),
+            op::movi(0x14, 100),
+            op::call(0x10, RegId::ZERO, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]);
+        replace_contract_in_service(&mut service, &contract_id, contract);
+        run_with_service(
+            "contract/tr",
+            group,
+            instructions,
+            script_data.clone(),
+            &service,
+            contract_id,
+            &rt,
+            &mut rng,
+        );
+    }
+
+    // tro
+
+    // The `tro` benchmark is disabled because it would require many, many outputs, because each
```

Comment:
Actually, we can still run this benchmark that will fail later because of the `255` limit to get at least some results. You can customize the `run` function to allow other error

Reply:
I've addressed all the concerns except this one. My `tro` code is failing with an error like `InvalidOperation` or something like that, so I commented it out for now.

Reply:
We can leave it for later=)